src/openrct2-ui/windows/About.cpp

```diff
@@ -47,19 +47,19 @@ enum WINDOW_ABOUT_WIDGET_IDX {
 
 #define WIDGETS_MAIN \
     WINDOW_SHIM(WINDOW_TITLE, WW, WH), \
-    { WWT_IMGBTN,           1,  0,              WW - 1,         TABHEIGHT,  WH - 1,     0xFFFFFFFF,                             STR_NONE },             /* page background */ \
-    { WWT_TAB,              1,  3,              93,             17,         TABHEIGHT,  IMAGE_TYPE_REMAP | SPR_TAB_LARGE,       STR_NONE },             /* about OpenRCT2 button */ \
-    { WWT_TAB,              1,  94,             184,            17,         TABHEIGHT,  IMAGE_TYPE_REMAP | SPR_TAB_LARGE,       STR_NONE }              /* about RCT2 button */
+    MakeWidget     ({ 0, TABHEIGHT}, {WW, WH - TABHEIGHT}, WWT_IMGBTN, 1               ), /* page background */       \
+    MakeRemapWidget({ 3,        17}, {91,             34}, WWT_TAB,    1, SPR_TAB_LARGE), /* about OpenRCT2 button */ \
+    MakeRemapWidget({94,        17}, {91,             34}, WWT_TAB,    1, SPR_TAB_LARGE)  /* about RCT2 button */
 
 static rct_widget window_about_openrct2_widgets[] = {
     WIDGETS_MAIN,
-    { WWT_BUTTON,           1,  100,            299,            WH - 50,    WH - 39,    STR_CHANGELOG_ELLIPSIS,                 STR_NONE },             // changelog button
+    MakeWidget({100, WH - TABHEIGHT}, {200, 12}, WWT_BUTTON, 1, STR_CHANGELOG_ELLIPSIS), // changelog button
```

Comment:
Not using `WH`

Reply:
We are for the position, not for the height. Here, again, WH is 350, not 200. Do you think `WH - 150` is more helpful? Personally, I don't think it's clearer…

Reply:
I think the current constants are fine for positioning, however, I think it's best to come up with new constants for their dimensions. As such, we are still using `WH` for the widget position, just not for the height.